### ISSUE TITLE
Message font size

### ISSUE THREAD

**Issue by mesutd0nmez** [mhmzdev/awesome_snackbar_content#16]
When displaying the message, i would like to set font-size too.

**Comment by JohnF17** [comment]
Hey @mesutd0nmez, I've submitted a PR that'll allow you to do just that. Currently waiting for it to get merged.
If its an urgent matter try using this until
```
awesome_snackbar_content: 
  git:
    url: https://github.com/JohnF17/awesome_snackbar_content
    ref: main
```

**Comment by mhmzdev** [comment]
Fixed in `0.1.3`
